imports/ui/components/EditList.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import { connect } from "react-redux";
+import { deleteOneCheckList } from "../actions/AppActions.js";
+
+class DeleteOne extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false
+    };
+  }
+
+  handleSubmit = event => {
+    event.preventDefault();
```

Comment:
After event prevent line, add a mechanism to check if item being deleted is last item on the Shopping List. If it is last item, call DeleteAll() instead of RemoveOne()